Editor/EditorCore/MenuAction.cs

```diff
@@ -175,12 +178,18 @@ protected virtual Texture2D disabledIcon
         public virtual int toolbarPriority { get { return -1; } }
 
         /// <summary>
-        /// Gets the icon to display on the toolbar for this action.
+        /// Gets the icon to display in the Context Menu for this action.
+        /// </summary>
+        public abstract Texture2D icon { get; }
+
+        /// <summary>
+        /// Gets the high resolution icon to display on the toolbar for this action.
         /// </summary>
         /// <remarks>
         /// This property is not used when the [Toolbar display mode](../manual/toolbar.html#toolbar-display-modes) is set to text.
         /// </remarks>
-        public abstract Texture2D icon { get; }
+
+        public virtual Texture2D icon2x { get { return null; } }
```

Comment:
shouldn't this be handled by the icon loading code depending on screen resolution? are we using the 2x size and 1x simultaneously?

Reply:
I changed this back! :D